app/build.gradle

```diff
@@ -27,6 +27,10 @@ android {
         release {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            buildConfigField 'Boolean', 'enableCrashlytics', 'true'
```

Comment:
Is camel case for buildconfig field common practice? Since it's a constant

Reply:
I believe that's the method name for creating a constant. [Take a look here](https://developer.android.com/studio/build/gradle-tips#simplify-app-development).

Reply:
Well yes. I meant the casing for "enableCrashlytics"